### PR TITLE
Add image-text-to-text + multimodal fixes

### PR DESCRIPTION
**Pull request by osanseviero**
Tentatively

* We align better what multimodal is
* We add a new type for llava

### CODE REVIEW

File: packages/tasks/src/pipelines.ts [hunk 1]
```diff
@@ -589,9 +589,15 @@ export const PIPELINE_DATA = {
 	},
 	"text-to-video": {
 		name: "Text-to-Video",
-		modality: "multimodal",
+		modality: "cv",
```
Comment (SBrandeis) [inline packages/tasks/src/pipelines.ts:592]: Same comment here

Comment (osanseviero) [reply packages/tasks/src/pipelines.ts:592]: Check the discussion in https://huggingface.slack.com/archives/C01N44FJDHT/p1706796001731059?thread_ts=1706792789.298009&cid=C01N44FJDHT

This model only has one type of model input (same as text-to-speech).

Comment (SBrandeis) [reply packages/tasks/src/pipelines.ts:592]: That makes sense, thank you for the link!

**Review by SBrandeis — APPROVED** [review]

**Review by Wauplin — APPROVED** [review]

File: packages/tasks/src/pipelines.ts [hunk 2]
```diff
@@ -430,7 +430,7 @@ export const PIPELINE_DATA = {
 				name: "Image Captioning",
 			},
 		],
-		modality: "multimodal",
+		modality: "cv",
```
Comment (amyeroberts) [inline packages/tasks/src/pipelines.ts:433]: Should this be "nlp"? The other labels seem to match the output modality.

Comment (osanseviero) [reply packages/tasks/src/pipelines.ts:433]: Hmm that would be a bit strange IMO. Both image generation and image captioning seem fundamentally computer vision tasks to me

**Comment by Reeray** [comment]
Hi there, here is an image-text-to-text icon suggestion: 
Image-text-to-text

three other icon suggestions for better icon consistency:
1. Image-to-text, 
image-to-text

2. Image-to-video, 
image-to-video

3. text-to-video,
text-to-video

**Review by SBrandeis — APPROVED** [review]

**Pull request merged**